src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/Cas1CruManagementAreaEntity.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity
+
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import org.hibernate.annotations.CacheConcurrencyStrategy
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+import java.util.UUID
+
+@Repository
+interface Cas1CruManagementAreaRepository : JpaRepository<Cas1CruManagementAreaEntity, UUID>
+
+/**
+ * When an application is submitted it will be assigned to
+ * a CRU Management Area. This will then be used to determine
+ * which CRU Members should manage assessment and matching
+ * for the application
+ */
+@Entity
+@Table(name = "cas1_cru_management_areas")
+@org.hibernate.annotations.Cache(usage = CacheConcurrencyStrategy.READ_ONLY)
+data class Cas1CruManagementAreaEntity(
+  @Id
+  val id: UUID,
+  val name: String,
+  val emailAddress: String?,
+  val notifyReplyToEmailId: String?,
+) {
+  companion object {
```

Comment:
Oh I have not come across these before, this seems neat

Reply:
i'm not sure if you're too familiar with java but it's the equivalent of 

`public static final UUID WOMENS_ESTATE_ID = ...;`

It's more clunky but i get why kotlin designers took this approach